src/skeletons.tsx

```diff
@@ -0,0 +1,184 @@
+// https://www.figma.com/file/w7E0mmB92eio0zHw7h9iS2/%5BREADY%5D-Skeletons-Specs?node-id=986%3A1161
+import * as React from 'react';
+import SkeletonBase from './skeleton-base';
+import {createUseStyles} from './jss';
+import Stack from './stack';
+import classNames from 'classnames';
+
+type AnimationProps = {
+    disableAnimation: boolean;
+};
+
+const transition = '1.5s linear';
+
+const useAnimation = createUseStyles(() => ({
+    animation: {
+        animation: ({disableAnimation}: AnimationProps) =>
+            disableAnimation ? '' : `$pulse ${transition} infinite`,
+    },
+    '@keyframes pulse': {
+        '0%': {
+            opacity: 1,
+        },
+
+        '50%': {
+            opacity: 0.5,
+        },
+
+        '100%': {
+            opacity: 1,
+        },
+    },
+}));
+
+type SkeletonStylesProps = {
+    height: number | string;
+    width: number | string;
+    isInverse: boolean;
+};
+
+const useSkeletonStyles = createUseStyles(() => ({
+    wrap: {
+        borderRadius: 8,
+        height: ({height = 8}: SkeletonStylesProps) => height,
+        width: ({width = 'content-initial'}) => width,
+    },
+}));
+
+type SkeletonLineProps = {
+    width?: string | number;
+    disableAnimation?: boolean;
+    ariaLabel?: string;
+};
+
+const SkeletonLine = ({
```

Comment:
please, add the dataAttributes prop to skeleton components (see other components to know how)